fme.md

```diff
@@ -4,495 +4,532 @@ N/A means "not applicable" in this context
-[VoronoiDiagrammer](https://www.safe.com/transformers/voronoi-diagrammer/)|Voronoi polygons|N/A|
-[WebMapTiler](https://www.safe.com/transformers/web-map-tiler/)|Generate XYZ tiles|N/A|
-[WebSocketReceiver](https://www.safe.com/transformers/web-socket-receiver/)|Missing|N/A|
-[WebSocketSender](https://www.safe.com/transformers/web-socket-sender/)|Missing|N/A|
-[WorkspaceRunner](https://www.safe.com/transformers/workspace-runner/)|N/A|N/A|
-[XMLAppender](https://www.safe.com/transformers/xml-appender/)|Missing|N/A|
-[XMLFeatureMapper](https://www.safe.com/transformers/xml-feature-mapper/)|Missing|N/A|
-[XMLFlattener](https://www.safe.com/transformers/xml-flattener/)|Missing|N/A|
-[XMLFormatter](https://www.safe.com/transformers/xml-formatter/)|Missing|N/A|
-[XMLFragmenter](https://www.safe.com/transformers/xml-fragmenter/)|Missing|N/A|
-[XMLNamespaceDeclarer](https://www.safe.com/transformers/xml-namespace-declarer/)|Missing|N/A|
-[XMLSampleGenerator](https://www.safe.com/transformers/xml-sample-generator/)|Missing|N/A|
-[XMLTemplater](https://www.safe.com/transformers/xml-templater/)|Missing|N/A|
-[XMLUpdater](https://www.safe.com/transformers/xml-updater/)|Missing|N/A|
-[XMLValidator](https://www.safe.com/transformers/xml-validator/)|Missing|N/A|
-[XMLXQueryExploder](https://www.safe.com/transformers/xmlx-query-exploder/)|Missing|N/A|
-[XMLXQueryExtractor](https://www.safe.com/transformers/xmlx-query-extractor/)|Missing|N/A|
-[XMLXQueryUpdater](https://www.safe.com/transformers/xmlx-query-updater/)|Missing|N/A|
+[2DArcReplacer](https://www.safe.com/transformers/2d-arc-replacer/)|N/A|Needs expression functions to create circular strings
+[2DBoxReplacer](https://www.safe.com/transformers/2d-box-replacer/)|N/A|Needs expression functions to create circular strings
+[2DEllipseReplacer](https://www.safe.com/transformers/2d-ellipse-replacer/)|N/A|Needs expression functions to create circular strings
+[2DForcer](https://www.safe.com/transformers/2d-forcer/)|Drop Z Value|Missing
+[2DGridAccumulator](https://www.safe.com/transformers/2d-grid-accumulator/)|Create grid (with extend option)|N/A
+[2DGridCreator](https://www.safe.com/transformers/2d-grid-creator/)|Create grid|N/A
+[3DAffiner](https://www.safe.com/transformers/3d-affiner/)|Affine transform|N/A
+[3DArcReplacer](https://www.safe.com/transformers/3d-arc-replacer/)|Missing|N/A
+[3DForcer](https://www.safe.com/transformers/3d-forcer/)|Set Z Value|Missing
+[3DInterpolator](https://www.safe.com/transformers/3d-interpolator/)|Missing|N/A|Also desirable for m values|
+[3DRotator](https://www.safe.com/transformers/3d-rotator/)|Rotate|N/A
+[Affiner](https://www.safe.com/transformers/affiner/)|Affine transform|Missing
+[AffineWarper](https://www.safe.com/transformers/affine-warper/)|Missing|N/A|Would use gdal to perform warp via gcps -- could expose more than just affine transformation. Possibly aided by Lutra's upcoming work improving the QGIS georeferencer.|
```

Comment:
FYI these classes where all moved to QGIS analysis lib to facilitate a possible future affine warper algorithm